ansible/configs/ansible-multitier-infra/sample_vars/configured_tower_osp.yml

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
Just a small point for both this and the ec2 sample_vars. These files should be what people need to use and make small changes to in order to deploy. It seems like this is pretty much all of default_vars. We should be able to deploy a config with 90% of the vars coming from default_vars and 10% that we have to customize in sample_vars, secrets, etc.

I made those percentages up, but you get the point.